src/main/java/org/fogbowcloud/manager/core/models/orders/Order.java

```diff
@@ -126,24 +141,35 @@ public long getFulfilledTime() {
 	public void setFulfilledTime(Long fulfilledTime) {
 		this.fulfilledTime = fulfilledTime;
 	}
-	
-	public boolean isLocal() {
-		return this.providingMember.equals(this.requestingMember);
+
+	public boolean isLocal(String localMemberId) {
+		return this.providingMember.equals(localMemberId);
 	}
-	
-	public boolean isRemote() {
-		return !this.providingMember.equals(this.requestingMember);
+
+	public boolean isRemote(String localMemberId) {
+		return !this.providingMember.equals(localMemberId);
+	}
+
+	/**
+	 * These method handle and request an open order, for this, processOpenOrder
```

Comment:
remove this process stuff

Reply:
Done, [here](https://github.com/fogbow/fogbow-manager-core/pull/13/commits/1b89ceb788759a1a441719c8e7f35f70949e8a82).